applications/nfc/scenes/nfc_scene_read_card.c

```diff
@@ -30,7 +30,7 @@ bool nfc_scene_read_card_on_event(void* context, SceneManagerEvent event) {
             return true;
         }
     } else if(event.type == SceneManagerEventTypeTick) {
-        notification_message(nfc->notifications, &sequence_blink_blue_10);
+        notification_message(nfc->notifications, &sequence_blink_cyan_10);
```

Comment:
May be change notifications also in Mifare Ultralight, Classic, Desfire and Bank Card read scenes?

Reply:
I will return this change back, cause it's a part of different task.